docs/azure/sdk/authentication/index.md

```diff
@@ -8,57 +8,74 @@ ms.date: 08/02/2024
 
-## Recommended app authentication approach
+## Recommended approach for app authentication
 
-It's recommended that apps use token-based authentication rather than connection strings when authenticating to Azure resources. The [Azure Identity library](/dotnet/api/overview/azure/identity-readme?view=azure-dotnet&preserve-view=true) provides classes that support token-based authentication and allow apps to seamlessly authenticate to Azure resources whether the app is in local development, deployed to Azure, or deployed to an on-premises server.
+Token-based authentication is the recommended approach for authenticating apps to Azure, instead of using connection strings or key-based options. The [Azure Identity library](/dotnet/api/overview/azure/identity-readme?view=azure-dotnet&preserve-view=true) provides classes that support token-based authentication and allow apps to seamlessly authenticate to Azure resources whether the app running locally, hosted on Azure, or hosted on an on-premises server.
 
-The specific type of token-based authentication an app should use to authenticate to Azure resources depends on where the app runs and is shown in the following diagram:
+### Advantages of token-based authentication
+
+Token-based authentication offers the following advantages over connection strings:
+
+- Token-based authentication ensures only the specific apps intended to access the Azure resource are able to do so, whereas anyone or any app with a connection string can connect to an Azure resource.
+- Token-based authentication allows you to further limit Azure resource access to only the specific permissions needed by the app. This follows the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege). In contrast, a connection string grants full rights to the Azure resource.
+- When using a [managed identity](/entra/identity/managed-identities-azure-resources/overview) for token-based authentication, Azure handles administrative functions for you, so you don't have to worry about tasks like securing or rotating secrets. This makes the app more secure because there's no connection string or application secret that can be compromised.
+- The [Azure.Identity](https://www.nuget.org/packages/Azure.Identity) package acquires and manages Microsoft Entra tokens for you. This makes using token-based authentication as easy to use as a connection string.
+
+Use of connection strings should be limited to initial proof-of-concept apps or development prototypes that don't access production or sensitive data. In all other scenarios, use the token-based authentication classes available in the Azure Identity library to authenticate to Azure resources.
```

Comment:
This first sentence falls apart for a lot of the database services, where connection strings must be used. For example, accessing Azure SQL via Microsoft.Data.SqlClient.